### ISSUE TITLE
Supporting WCAG requirements.

### ISSUE THREAD

**Issue by Horiatu** [kepi/chromeEyeDropper#79]
I would extend this to support WCAG requirements for contrast.
Basically, the picker will store two colors (for foreground and background) and calculate the contrast ratio.

Not sure how to calculate the contrast but good resources are:
http://contrast-finder.tanaguru.com/ 
http://leaverou.github.io/contrast-ratio/
http://www.colorsontheweb.com/colorcontrast.asp
all above are missing a picker, also are not implemented as an extension.

(I would also be pleased to be able to call the extension from the configuration phase of my project assuming that the tool was intalled.)

**Comment by kepi** [comment]
That sounds interesting. Not sure about last part - how you want to call the extension exactly?

**Comment by kepi** [comment]
similar to #107

**Comment by Horiatu** [comment]
My initial comment is 7 years old.
In the mean time, I've built an extension to do that:
"WCAG Luminosity Contrast Ratio Analyzer" - https://chrome.google.com/webstore/detail/wcag-luminosity-contrast/lllpnmpooomecmbmijbmbikaacgfdagi
And later, Google developer tools has included some of the functionality I'm providing there.
However, my extension provides more goodies, suggesting alternate accessible color sets and names for appropriate colors. It also support filters to see a page in challenged visions (may not work anymore, sorry - some deprecated function in JS have invalidate this feature. The code is now many years old and, although I have upgraded it on my computer, Google Developer program does not accept it anymore, or require too much effort from me to re-deploy...)
Give it a try, tell me what you think.
